kivy/lang.py

```diff
@@ -1953,6 +1958,23 @@ def template(self, *args, **ctx):
         self._apply_rule(widget, rule, rule, template_ctx=proxy_ctx)
         return widget
 
+    def apply_rules(self, widget, widget_name, ignored_consts=set()):
```

Comment:
`rule_name`. rather than `widget_name` is a better choice. Also, you need to add the versionadded think, like here: https://github.com/udiboy1209/kivy/blob/styling/kivy/uix/label.py#L645. The version is 1.9.2.